packages/semantic-ui/src/CheckboxWidget/CheckboxWidget.js

```diff
@@ -17,8 +16,10 @@ function CheckboxWidget(props) {
     options,
     onFocus,
     formContext,
+    schema,
   } = props;
   const semanticProps = getSemanticProps({ formContext, options });
+  const desc = label || schema.description;
```

Comment:
Can we undo this change for this PR for now?

Normally, if the label and description are both specified, we'd want to show both instead of just overriding the description with the label (https://github.com/rjsf-team/react-jsonschema-form/blob/b78d9ef280eddf5bda0f97e5a3445c6a1fd35c99/packages/core/src/components/widgets/CheckboxWidget.js#L27-L42). However, I think that might be beyond the scope of this PR and we can make another issue / PR addressing it.